api/src/main/resources/org/openmrs/module/dtd/config-1.6.dtd

```diff
@@ -88,7 +88,7 @@
 
 	<!ELEMENT signatures (#PCDATA)>
 
-	<!ELEMENT servlet (servlet-name, servlet-class)>
+	<!ELEMENT servlet (servlet-name, servlet-class, init-param*)>
```

Comment:
Shouldn't we create `config-1.7.dtd` instead of modifying this one?

Reply:
Yes sure, updated with the new `config-1.7.dtd` and updated tests to validate backwards-compatibility